remix/.env.sample

```
@@ -0,0 +1 @@
+DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public"
```

How about using the default credentials define in Docker Compose (it would remove any editing need):

```suggestion
DATABASE_URL="postgresql://postgres:postgres@localhost:5432/mydb?schema=public"
```